migrations/V183__Add_MTO_Common_Solution.sql

```diff
@@ -1,26 +1,37 @@
 CREATE TYPE MTO_COMMON_SOLUTION_KEY AS ENUM (
-    'SOLUTION_A',
-    'SOLUTION_B'
+    'INNOVATION', 'ACO_OS', 'APPS', 'CDX', 'CCW', 'CMS_BOX', 'CMS_QUALTRICS', 'CBOSC', 'CPI_VETTING', 'EFT', 'EXISTING_CMS_DATA_AND_PROCESS', 'EDFR', 'GOVDELIVERY', 'GS', 'HDR', 'HPMS', 'HIGLAS', 'IPC', 'IDR', 'LDG', 'LV', 'MDM_POR', 'MARX', 'OUTLOOK_MAILBOX', 'QV', 'RMADA', 'ARS', 'CONNECT', 'LOI', 'POST_PORTAL', 'RFA', 'SHARED_SYSTEMS', 'BCDA', 'ISP', 'MIDS', 'MDM_NCBP', 'MODEL_SPACE'
 );
-
 CREATE TYPE MTO_SOLUTION_TYPE AS ENUM (
     'IT_SYSTEM',
     'CONTRACT',
     'CROSS_CUTTING_GROUP',
     'OTHER'
 );
+
+CREATE TYPE MTO_COMMON_SOLUTION_SUBJECT AS ENUM (
+    'APPLICATIONS_AND_PARTICIPANT_INTERACTION_ACO_AND_KIDNEY_MODELS',
+    'APPLICATIONS_AND_PARTICIPANT_INTERACTION_NON_ACO_MODELS',
+    'MEDICARE_FEE_FOR_SERVICE',
+    'CONTRACT_VEHICLES',
+    'LEARNING',
+    'QUALITY',
+    'COMMUNICATION_TOOLS_AND_HELP_DESK',
+    'MEDICARE_ADVANTAGE_AND_PART_D',
+    'PAYMENTS_AND_FINANCIALS',
+    'DATA',
+    'LEGAL'
+);
+
 COMMENT ON TYPE MTO_SOLUTION_TYPE IS
 'Specifies the type of solution, such as IT system, contract, cross-cutting group, or other, to categorize solutions used within the model plan.';
 
 
 CREATE TABLE mto_common_solution (
-    -- id UUID PRIMARY KEY,
     key MTO_COMMON_SOLUTION_KEY PRIMARY KEY,
     name ZERO_STRING NOT NULL,
-
     type MTO_SOLUTION_TYPE NOT NULL,
-    description ZERO_STRING NOT NULL,
-    facilitated_by_role MTO_FACILITATOR[] NOT NULL
+    subject MTO_COMMON_SOLUTION_SUBJECT[] NOT NULL,
+    filter_view MODEL_VIEW_FILTER

```

Comment:
Thanks @ClayBenson94 ! Happy to sync on this. It is a direct migration of the field from operational solutions. This field is used specifically when a solution is selected and an email is sent to the points of contact. When they receive an email, if this is populated they will be linked to the read view with this filter applied (hence why it is only at most 1). Zoë and I chatted a little about this, the FIGMA for the notification hasn't been created yet though

Reply:
Understood -- that makes total sense for why it would be just a single filter view here! 

I was assuming it was something about which solutions we'd show when you select a specific filter view, but I think this clarifies things!

With the comment in place now that helps settle this!